databricks-sdk-java/src/main/java/com/databricks/sdk/core/ApiClient.java

```diff
@@ -276,10 +276,12 @@ private Response executeInner(Request in) {
       // Retry after a backoff.
       long sleepMillis = getBackoffMillis(out, attemptNumber);
       LOG.debug(String.format("Retry %s in %dms", in.getRequestLine(), sleepMillis));
-      try {
-        Thread.sleep(sleepMillis);
-      } catch (InterruptedException ex) {
-        Thread.currentThread().interrupt();
+      synchronized (timer) {
+        try {
+          timer.wait(sleepMillis);
+        } catch (InterruptedException ex) {
+          Thread.currentThread().interrupt();
+        }
```

Comment:
This issue is probably because `wait()` is actually inherited from `Object` and is used for synchronization. Let's rename this method to `sleep()`, then we won't need this here.

Reply:
Ah, my bad, I hadn't realised timer was actually defined in the sdk